src/js/components/search/filters/location/LocationSearch.jsx

```diff
@@ -4,19 +4,90 @@
  **/
 
 import React from 'react';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import _ from 'lodash';
+
+import * as SearchHelper from 'helpers/searchHelper';
+import * as autocompleteActions from 'redux/actions/search/autocompleteActions';
+
 import LocationListContainer from 'containers/sharedContainers/LocationListContainer';
 import CountryType from './CountryType';
 import SelectedLocations from './SelectedLocations';
 
 const propTypes = {
-    toggleCountry: React.PropTypes.func,
     selectLocation: React.PropTypes.func,
     removeLocation: React.PropTypes.func,
-    selectedLocations: React.PropTypes.object,
-    locationOption: React.PropTypes.string
+    setAutocompleteLocations: React.PropTypes.func,
+    updateDomesticForeignSelection: React.PropTypes.func,
+    selectedLocations: React.PropTypes.object
 };
 
-export default class LocationSearch extends React.Component {
+class LocationSearch extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            locationOption: 'all',
```

Comment:
Since Redux now tracks the `locationDomesticForeign` value, the autocomplete location scope should be controlled by Redux (ie, read from the props, write via the action rather than maintain its own state).